packages/ssx-react/src/next/frontend.ts

```diff
@@ -0,0 +1,31 @@
+import { getCsrfToken, signIn, signOut } from 'next-auth/react';
+import { SSXClientSession } from '@spruceid/ssx';
+
+export const SSXNextAuthRouteConfig = () => {
```

Comment:
This is Approach A: Providing a configuration using custom operations to call `NextAuth`'s config. If we went with approach B, we would still use this configuration provider, but would call no-ops (`customOperation: () => {}`) here.